templates/common/.devcontainer/devcontainer.json/python/terraform/devcontainer.json

```diff
@@ -1,35 +1,43 @@
 {
-  "name": "Azure Developer CLI",
-  "build": {
-    "dockerfile": "Dockerfile",
-    "args": {
-      "VARIANT": "bullseye"
-    }
-  },
-  "features": {
-    "github-cli": "2",
-    "azure-cli": "2.38",
-    "python": "os-provided",
-    "docker-from-docker": "20.10",
-    "node": {
-      "version": "16",
-      "nodeGypDependencies": false
+    "name": "Azure Developer CLI",
```

Comment:
I fixed the spacing on this from 2 to 4 spaces to be consistent with the other files